test/mongooseim_metrics_SUITE.erl

```diff
@@ -91,6 +105,15 @@ tcp_connections_detected(_C) ->
     {ok, [{value, X}]} = mongoose_metrics:get_metric_value(global, tcpPortsUsed),
     ?assert(X > 0).
 
+queued_messages_increase(_C) ->
```

Comment:
This is my small test. It is making up some processes and sending them messages that won't match. We know how many processes and how many messages, so we know the minimum value of the message queue length, which is what we test. Is that enough? It seems to me that we can test if this decreases after the processes die, in the same test or in a separate one, but I don't know if that adds any value to the testing suite.